adal/tests/Test.ADAL.NET.Common/TestConstants.cs

```diff
@@ -58,6 +58,7 @@ public class TestConstants
         public static readonly string UserRealmEndPoint = "userrealm";
         public static readonly string DiscoveryEndPoint = "discovery/instance";
         public static readonly string DefaultRefreshTokenValue = "RefreshTokenValue";
+        public static readonly string DefaultAuthorizationCode = "AwABAAAAvPM1KaPlrEqdFSBzjqfTGBCmLdgfSTLEMPGYuNHSUYBrqqf_ZT_p5uEAEJJ_nZ3UmphWygRNy2C3jJ239gV_DBnZ2syeg95Ki-374WHUP-i3yIhv5i-7KU2CEoPXwURQp6IVYMw-DjAOzn7C3JCu5wpngXmbZKtJdWmiBzHpcO2aICJPu1KvJrDLDP20chJBXzVYJtkfjviLNNW7l7Y3ydcHDsBRKZc3GuMQanmcghXPyoDg41g8XbwPudVh7uCmUponBQpIhbuffFP_tbV8SNzsPoFz9CLpBCZagJVXeqWoYMPe2dSsPiLO9Alf_YIe5zpi-zY4C3aLw5g9at35eZTfNd0gBRpR5ojkMIcZZ6IgAA";
```

Comment:
>DefaultAuthorizationCode [](start = 38, length = 24)

shouldn't this be in keyvault?

Reply:
it is a dummy code that I randomly generated and it is just for testing the api. It is never sent to any endpoints as it is intercepted by the mock handlers.

Reply:
It may though be though of by a tool as an important key... not entirely sure how to reveal that intent, however perhaps if you add that to the name of the test constant (e.g. DummyAuthorizationCode)

Reply:
Can you randomly generate this one each run as part of the test code?

---
In reply to: [201777626](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r201777626) [](ancestors = 201777626)

Reply:
Yes, we can do that